### PR TITLE
ureport: Drop Strata integration

### PR DESCRIPTION
**Pull request by ernestask**
Strata is being sunset with no replacement for our used API endpoints in
sight. This commit drops the integration code, because no one pretty
much wants it, either.

### CODE REVIEW

**Comment by ernestask** [comment]
Requesting some reviews, because this has stalled. I’ll get on the conflicts in the meantime.

**Comment by michalfabik** [comment]
Nitpick: `tests/ureport/rhsm/__pycache__/*` isn't needed in `.gitignore` any more.

File: src/include/ureport.h [hunk 1]
```diff
@@ -111,7 +111,7 @@ libreport_ureport_server_config_set_url(struct ureport_server_config *config,
  *
  * @param config Where the paths are stored
  * @param client_path Path in form of cert_full_path:key_full_path or one of
- *        the following string: 'rhsm', 'puppet'.
+ *        the following string: 'puppet'.
```
Comment (mgrabovsky) [inline src/include/ureport.h:114]: "or one of the following string" 🤔

Comment (ernestask) [reply src/include/ureport.h:114]: The string was ahead of its time.

**Comment by ernestask** [comment]
Addressed both comments, going to proceed with the merge.

**Pull request merged**